cardano-node/src/Cardano/Node/Configuration/Logging.hs

```diff
@@ -166,8 +166,8 @@ createLoggingLayer ver nodecli@NodeCLI{configFile} = do
      -> Trace IO Text
      -> IO ()
    loggingPreInit nodeConfig logConfig switchBoard trace = do
-     Config.getEKGport logConfig >>= \p ->
-       when (p > 0) $
+     Config.getEKGBindAddr logConfig >>= \mbEndpoint ->
+       when (isJust mbEndpoint) $
```

Comment:
@denisshevchenko This changed in https://github.com/input-output-hk/iohk-monitoring-framework/pull/570, is this correct?

Reply:
@mrBliss Yes, it is correct: https://github.com/input-output-hk/iohk-monitoring-framework/pull/570/files#diff-a4a2e19aead2886583c9bee71a24fbacR300